www/main/settings/components/ThemeContainer.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import { remote } from 'electron';
+import { Tile } from '../../common/sttm-ui';
+
+import { themes } from '../../theme_editor';
+
+import { useStoreActions, useStoreState } from 'easy-peasy';
+
+const { i18n, store } = remote.require('./app');
+const analytics = remote.getGlobal('analytics');
+
+const themeTypes = [
+  { type: 'COLOR', title: 'COLORS' },
+  { type: 'BACKGROUND', title: 'BACKGROUNDS' },
+  { type: 'SPECIAL', title: 'SPECIAL_CONDITIONS' },
+];
+
+const ThemeContainer = () => {
+  const { theme: currentTheme } = useStoreState(state => state.userSettings);
+  const { setTheme } = useStoreActions(state => state.userSettings);
+
+  const applyTheme = (themeInstance, isCustom) => {
```

Comment:
Might need to pull the hot fix we did into the branch, don't think settings are applying properly to extended display.